src/Microsoft.Azure.WebJobs.Host/Bindings/BindingTemplateExtensions.cs

```diff
@@ -46,23 +46,15 @@ public static void ValidateContractCompatibility(this BindingTemplate bindingTem
         /// <param name="bindingTemplate">The binding template to validate.</param>
         /// <param name="bindingData">The binding data to apply to the template.</param>
         /// <returns>The bound template string.</returns>
+        [Obsolete("Call instance method directly")]
         public static string Bind(this BindingTemplate bindingTemplate, IReadOnlyDictionary<string, object> bindingData)
```

Comment:
All the extensions in the extensions repo call this currently. It looks like you've made the method you're directing to internal so extensions won't be able to call it.